docs/contracts/inheritance.rst

```diff
@@ -51,6 +51,8 @@ Details are given in the following example.
     // contracts can access all non-private members including
     // internal functions and state variables. These cannot be
     // accessed externally via `this`, though.
+    // The keyword `virtual` means that the function can change
+    // its behaviour in derived classes ("overriding").
     contract Mortal is Owned {
```

Comment:
The newly added comments (2 lines) can be moved here right before function definition